examples/go/add-order/main.go

```diff
@@ -20,7 +20,7 @@ import (
 
 type clientEnvVars struct {
 	// RPCAddress is the address of the 0x Mesh node to communicate with.
-	RPCAddress string `envvar:"RPC_ADDRESS"`
+	WSRPCAddress string `envvar:"WS_RPC_ADDRESS"`
```

Comment:
Let's take the opportunity to make this the same env var name as in Mesh itself.
```suggestion
	WSRPCAddress string `envvar:"WS_RPC_ADDR"`
```